src/plugins/saved_objects_tagging_oss/common/types.ts

```diff
@@ -19,10 +19,14 @@ export interface TagAttributes {
   color: string;
 }
 
+export interface GetAllTagsOptions {
+  asSystemRequest?: boolean;
+}
+
 export interface ITagsClient {
   create(attributes: TagAttributes): Promise<Tag>;
   get(id: string): Promise<Tag>;
-  getAll(): Promise<Tag[]>;
+  getAll(options?: GetAllTagsOptions): Promise<Tag[]>;
```

Comment:
This interface is also used by the server-side

https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/x-pack/plugins/saved_objects_tagging/server/services/tags/tags_client.ts#L19

and I think this option doesn't make any sense on the server.

But refactoring the code to use distinct interface is probably going to be a pain, so I'm ok keeping these changes for now as we do want that for `7.12`.